### ISSUE TITLE
App fails to start if first time run is on boot

### ISSUE THREAD

**Issue by t1m0thyj** [t1m0thyj/WinDynamicDesktop#199]
Reported by @RussellJQA

**Comment by t1m0thyj** [comment]
![image](https://user-images.githubusercontent.com/22344007/80265866-13ca4d00-8667-11ea-83b2-9e12e7b006c9.png)
```
Problem signature:
  Problem Event Name:	CLR20r3
  Problem Signature 01:	WinDynamicDesktop.exe
  Problem Signature 02:	4.0.0.0
  Problem Signature 03:	5e4b5f14
  Problem Signature 04:	mscorlib
  Problem Signature 05:	4.8.4121.0
  Problem Signature 06:	5deacd98
  Problem Signature 07:	1627
  Problem Signature 08:	0
  Problem Signature 09:	System.FormatException
  OS Version:	6.3.9600.2.0.0.256.48
  Locale ID:	2057
  Additional Information 1:	5861
  Additional Information 2:	5861822e1919d7c014bbb064c64908b2
  Additional Information 3:	6fbe
  Additional Information 4:	6fbe6bde2701766d81cbca0597a5fa35

Read our privacy statement online:
  http://go.microsoft.com/fwlink/?linkid=280262

If the online privacy statement is not available, please read our privacy statement offline:
  C:\Windows\system32\en-US\erofflps.txt
```

**Comment by shazr** [comment]
So this happened to me and it turned out it had to do with the "lastShuffleDate" on the config file. The long date format on my PC had been set to have spaces instead of colons, and this seem to have caused the issue. Changing the date format to include colons fixed it for me. But it would be nice if the program recognized the format and handled it accordinly.

**Comment by t1m0thyj** [comment]
@RussellJQA Unable to reproduce this issue. I did a clean install of WDD without starting it yet, then restarted and WDD launched automatically without any errors. Feel free to reopen if you can provide more details that I missed.

**Comment by sha-razeek** [comment]
Hi t1m0thyj, I'm afraid that error is legit. Would you please be able to change your region settings to something like 'Sinhala (Sri Lanka)' and try again please. That region uses . instead of : for the time and WDD crashes everytime I open it.

**Comment by t1m0thyj** [comment]
@sha-razeek If WDD is failing to start because of how "lastShuffleDate" is formatted, that is a different error (#207). Please make sure you're using the latest version of WDD (4.2.0). Errors with date parsing should have been fixed in v4.1.1. If the error still occurs, remove all dates from the "settings.conf" file and restart the app.

**Comment by sha-razeek** [comment]
Oh yes. Thanks! Works like a charm now!
